docs/github.md

```diff
@@ -0,0 +1,41 @@
+# Using Jujutsu with Github Projects
+
+## Setup a ssh key
+As of December 2022 it's necessary to setup a ssh key to work with github
```

Comment:
I thought HTTP authentication with password would also be possible since https://github.com/martinvonz/jj/pull/715. Is that not the case?

I think it's still a good idea to recommend using `ssh-agent`. What do you think about suggesting that here?

Reply:
I didn't test it. But when I was pushing a update to this pr, it popped up.